infra/ansible/inventory.yml

```diff
@@ -126,6 +128,7 @@ all:
           django_database_name: tournesol
           django_database_user: tournesol
           django_email_backend: smtp
+          django_enable_api_wikidata: true
```

Comment:
I think this name can be misleading: even with `djano_enable_api_wikidata: false`, requests to Wikidata can be triggered from admin UI. To rename to show that it applies on migrations only, or update the implementation to apply this value both in migrations and on entity refresh.

Reply:
oops, I'll mention the migrations here.